digitalforms-api/src/main/java/ca/bc/gov/open/pssg/rsbc/digitalforms/model/DocumentInfo.java

```diff
@@ -0,0 +1,48 @@
+package ca.bc.gov.open.pssg.rsbc.digitalforms.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+/**
+ * 
+ * Document object
+ * 
+ * @author sivakaruna
+ *
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({ "mime", "document" })
+public class DocumentInfo {
+
+	@JsonProperty("mime")
```

Comment:
maybe change this to "mimeType". Thanks.

Reply:
updated